contrib/cirrus/runner.sh

```diff
@@ -269,7 +261,7 @@ function _run_altbuild() {
         *Windows*)
 	    showrun make .install.pre-commit
             showrun make lint GOOS=windows CGO_ENABLED=0
-            showrun make podman-remote-release-windows_amd64.zip
+            showrun make podman-remote-release-windows_amd64.zip validate-binaries
```

Comment:
I Don't think this is designed to be run for windows and cross builds, neither of those would actually validate the cross build binary and instead build the normal binary again.

Reply:
oh good point.  Thanks.

Reply:
Mmmm, and in fact it's just checking man pages and completions, so probably no need to do this on OSX or the other alt-builds either.